django/core/checks/urls.py

```diff
@@ -28,6 +28,30 @@ def check_resolver(resolver):
         return []
 
 
+@register(Tags.urls)
+def check_url_namespaces_unique(app_configs, **kwargs):
+    """
+    Checks that all URL namespaces used in applications are unique.
+    """
+    from django.urls import get_resolver
```

Comment:
You should probably check `if getattr(settings, 'ROOT_URLCONF', None)`, similar to the `check_url_config()` check.